bundle/language_en.properties

```diff
@@ -1188,6 +1188,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents are together
 angal.patient.patientbrowser.title                                                                     = Patient Browser
 angal.patient.patientphoto                                                                             = Patient Photo
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound.msg                                                      = Patient photo file was not found
```

Comment:
Messages should be sentences and thus it should end in a period.   Better yet, `The patient photo file was not found.`

Reply:
also the title?

Reply:
No dialog titles are fine without the period